fastlane_bot/helpers/txhelpers.py

```diff
@@ -551,148 +507,60 @@ def _submit_transaction(self, tx_hash) -> str:
             )
             return None
 
-    def sign_transaction(self, transaction: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
see comment above: I would probably not remove this abstraction layer here, it may have been put in for a reason and may be useful

Reply:
No abstraction here. It probably used to be a larger function, which got smaller over time.
It also appears as an internal function in a few other modules, making it a little difficult to follow the code when searching for it.
Anyways, it was a single-line function at this point, so moved that line directly to where the function was called, in order to make the code easier to read.

Reply:
disagree. it abstracts from the exact use of the web3 library. maybe this is the right thing to do, but it is not self evident

Reply:
Noted.